### PR TITLE
Civ 640 citizen views claims defended

### PR DESCRIPTION
**Pull request by laarmada**
#609 CIV-640_Citizen_views_claims_defended
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/CIV-661

### Change description ###
Implement API call to show list of claims

**Does this PR introduce a breaking change?** (check one with "x")

```
[x] Yes
[ ] No
```

### CODE REVIEW

File: bin/utils/process-definition.sh [hunk 1]
```diff
@@ -14,9 +14,9 @@ if [[ ! -e ${definition_output_file} ]]; then
    touch ${definition_output_file}
 fi
 
-docker run --rm --name json2xlsx \
+docker run --rm --name json2xlsx --user=$UID\
   -v ${definition_input_dir}:/tmp/ccd-definition \
   -v ${definition_output_file}:/tmp/ccd-definition.xlsx \
-  -e CCD_DEF_CASE_SERVICE_BASE_URL=${CCD_DEF_CASE_SERVICE_BASE_URL:-http://docker.for.mac.localhost:4000} \
+  -e CCD_DEF_CASE_SERVICE_BASE_URL=${CCD_DEF_CASE_SERVICE_BASE_URL:-http://${DOCKERFORLINUX}:4000} \
```
Comment (HarryH96) [inline bin/utils/process-definition.sh:20]: Please revert before merge

Comment (laarmada) [reply bin/utils/process-definition.sh:20]: sure, thanks!

File: src/main/java/uk/gov/hmcts/reform/civil/controllers/cases/CasesController.java [hunk 2]
```diff
@@ -49,4 +56,19 @@
 
         return new ResponseEntity<>(caseDataResponse, HttpStatus.OK);
     }
+
+    @PostMapping(path = "/")
+    @ApiOperation("get list of the cases from CCD")
+
+    public ResponseEntity<SearchResult> getListCase(@RequestHeader(HttpHeaders.AUTHORIZATION) String authorization,
+                                                @RequestBody String searchString) {
```
Comment (mounikahmcts) [inline src/main/java/uk/gov/hmcts/reform/civil/controllers/cases/CasesController.java:64]: Is it getCaseList? method name

Comment (LeonardoPalmeiro) [reply src/main/java/uk/gov/hmcts/reform/civil/controllers/cases/CasesController.java:64]: done

**Review by sampankumar — APPROVED** [review]

**Pull request merged**